### PR TITLE
[Localization] Add localization hard-coded message in ability (AIR_LOCK, CLOUD_NINE)

### PR DESCRIPTION
**Pull request by EnochG1**
## What are the changes the user will see?
English : none.
Other languages : Can see translated messages.

## Why am I making these changes?
All message texts are localizable.

## What are the changes from a developer perspective?
 - Change hard-coded text for localize > src/data/ability.ts
 - Added localization keys in each language folders.
 - Translated them to Korean. (+ each language after review)

### Screenshots/Videos

| Lang | Case | Result |
| :-----: | :-----: | :-----: |
| En | CLOUD_NINE | image |
| Es | CLOUD_NINE | image |
| It | CLOUD_NINE | image |
| Fr | CLOUD_NINE | image |
| De | CLOUD_NINE | image |
| Pt_BR | CLOUD_NINE | image |
| Zh_CN | CLOUD_NINE | image |
| Zh_TW | CLOUD_NINE | image |
| Ko | CLOUD_NINE | image |
| Ja | CLOUD_NINE | image |

## How to test the changes?
Manually.
1. Checkout this PR and change language
2. set overrides.ts
```typescript
// choose one ability of these
ABILITY_OVERRIDE: Abilities = Abilities.AIR_LOCK;
ABILITY_OVERRIDE: Abilities = Abilities.CLOUD_NINE;
```
and start 'New Game' and choose any Pokemon.

## Checklist
- [x] **I'm using `beta` as my base branch**
- [x] There is no overlap with another PR?
- [x] The PR is self-contained and cannot be split into smaller PRs?
- [x] Have I provided a clear explanation of the changes?
- [ ] Have I considered writing automated tests for the issue?
- [x] If I have text, did I add placeholders for them in locales?
- [x] Have I tested the changes (manually)?
    - [x] Are all unit tests still passing? (`npm run test`)
- [x] Are the changes visual?
  - [x] Have I provided screenshots/videos of the changes?

### CODE REVIEW

File: src/locales/de/ability-trigger.ts [hunk 1]
```diff
@@ -46,6 +46,7 @@ export const abilityTriggers: SimpleTranslationEntries = {
   "postFaintContactDamage": "{{abilityName}} von {{pokemonNameWithAffix}} schadet seinem Angreifer!",
   "postFaintHpDamage": "{{abilityName}} von {{pokemonNameWithAffix}} schadet seinem Angreifer!",
   "postSummonPressure": "{{pokemonNameWithAffix}} setzt Gegner mit Erzwinger unter Druck!",
+  "weatherEffectDisappeared": "The effects of the weather disappeared.",
```
Comment (CodeTappert) [inline src/locales/de/ability-trigger.ts:49]:
```suggestion
  "weatherEffectDisappeared": "Jegliche wetterbedingten Effekte wurden aufgehoben!",
```

File: src/locales/zh_CN/ability-trigger.ts [hunk 2]
```diff
@@ -46,6 +46,7 @@ export const abilityTriggers: SimpleTranslationEntries = {
   "postFaintContactDamage": "{{pokemonNameWithAffix}}的{{abilityName}}\n使对方受到了伤害！",
   "postFaintHpDamage": "{{pokemonNameWithAffix}}的{{abilityName}}\n使对方受到了伤害！",
   "postSummonPressure": "从{{pokemonNameWithAffix}}的身上\n感到了一种压迫感！",
+  "weatherEffectDisappeared": "The effects of the weather disappeared.",
```
Comment (mercurius-00) [inline src/locales/zh_CN/ability-trigger.ts:49]:
```suggestion
  "weatherEffectDisappeared": "天气的影响消失了！",
```

File: src/locales/zh_TW/ability-trigger.ts [hunk 3]
```diff
@@ -46,6 +46,7 @@ export const abilityTriggers: SimpleTranslationEntries = {
   "postFaintContactDamage": "{{pokemonNameWithAffix}}'s {{abilityName}}\nhurt its attacker!",
   "postFaintHpDamage": "{{pokemonNameWithAffix}}'s {{abilityName}}\nhurt its attacker!",
   "postSummonPressure": "{{pokemonNameWithAffix}} is exerting its Pressure!",
+  "weatherEffectDisappeared": "The effects of the weather disappeared.",
```
Comment (mercurius-00) [inline src/locales/zh_TW/ability-trigger.ts:49]:
```suggestion
  "weatherEffectDisappeared": "天氣的影響消失了！",
```

**Review by mercurius-00 — COMMENTED** [review]
zh

File: src/locales/it/ability-trigger.ts [hunk 4]
```diff
@@ -46,6 +46,7 @@ export const abilityTriggers: SimpleTranslationEntries = {
   "postFaintContactDamage": "{{pokemonNameWithAffix}}'s {{abilityName}}\nhurt its attacker!",
   "postFaintHpDamage": "{{pokemonNameWithAffix}}'s {{abilityName}}\nhurt its attacker!",
   "postSummonPressure": "{{pokemonNameWithAffix}} is exerting its Pressure!",
+  "weatherEffectDisappeared": "The effects of the weather disappeared.",
```
Comment (NicusPulcis) [inline src/locales/it/ability-trigger.ts:49]:
```suggestion
  "weatherEffectDisappeared": "Le condizioni atmosferiche non hanno alcun effetto.",
```

**Review by NicusPulcis — APPROVED** [review]
italian localization done : )

**Comment by NicusPulcis** [comment]
btw @EnochG1 does this PR mess with the previous one i made for italian? (#3546 this one)

**Comment by EnochG1** [comment]
@NicusPulcis
I think your PR will be merged first so even though conflict occurs I can resolve it.

File: src/locales/fr/ability-trigger.ts [hunk 5]
```diff
@@ -46,6 +46,7 @@ export const abilityTriggers: SimpleTranslationEntries = {
   "postFaintContactDamage": "{{pokemonNameWithAffix}} est blessé\npar son talent {{abilityName}} !",
   "postFaintHpDamage": "{{pokemonNameWithAffix}} est blessé\npar son talent {{abilityName}} !",
   "postSummonPressure": "{{pokemonNameWithAffix}}\naugmente la pression !",
+  "weatherEffectDisappeared": "The effects of the weather disappeared.",
```
Comment (Adri1) [inline src/locales/fr/ability-trigger.ts:49]:
```suggestion
  "weatherEffectDisappeared": "Les effets de la météo se dissipent !",
```

**Comment by NicusPulcis** [comment]
@EnochG1 alright! thanks : )

**Review by Adri1 — APPROVED** [review]

File: src/locales/pt_BR/ability-trigger.ts [hunk 6]
```diff
@@ -46,6 +46,7 @@ export const abilityTriggers: SimpleTranslationEntries = {
   "postFaintContactDamage": "{{abilityName}} de {{pokemonNameWithAffix}}\nferiu seu adversário!",
   "postFaintHpDamage": "{{abilityName}} de {{pokemonNameWithAffix}}\nferiu seu adversário!",
   "postSummonPressure": "{{pokemonNameWithAffix}} está exercendo sua pressão!",
+  "weatherEffectDisappeared": "The effects of the weather disappeared.",
```
Comment (josericardo-fo) [inline src/locales/pt_BR/ability-trigger.ts:49]:
```suggestion
  "weatherEffectDisappeared": "Os efeitos do clima desapareceram.",
```

File: src/locales/ja/ability-trigger.ts [hunk 7]
```diff
@@ -47,6 +47,7 @@ export const abilityTriggers: SimpleTranslationEntries = {
   "postFaintContactDamage": "{{pokemonNameWithAffix}}は {{abilityName}}で\n相手に ダメージを 与えた！",
   "postFaintHpDamage": "{{pokemonNameWithAffix}}は　{{abilityName}}で\n相手に　ダメージを　与えた！",
   "postSummonPressure": "{{pokemonNameWithAffix}}は\nプレッシャーを　放っている！",
+  "weatherEffectDisappeared": "The effects of the weather disappeared.",
```
Comment (Adri1) [inline src/locales/ja/ability-trigger.ts:50]:
```suggestion
  "weatherEffectDisappeared": "天候の影響が　なくなった！",
```

Comment (Adri1) [reply src/locales/ja/ability-trigger.ts:50]: Added using Pokécorpus

File: src/locales/es/ability-trigger.ts [hunk 8]
```diff
@@ -46,6 +46,7 @@ export const abilityTriggers: SimpleTranslationEntries = {
   "postFaintContactDamage": "{{pokemonNameWithAffix}}'s {{abilityName}}\nhurt its attacker!",
   "postFaintHpDamage": "{{pokemonNameWithAffix}}'s {{abilityName}}\nhurt its attacker!",
   "postSummonPressure": "{{pokemonNameWithAffix}} is exerting its Pressure!",
+  "weatherEffectDisappeared": "The effects of the weather disappeared.",
```
Comment (AsdarDevelops) [inline src/locales/es/ability-trigger.ts:49]:
```suggestion
  "weatherEffectDisappeared": "El tiempo atmosférico ya no ejerce ninguna influencia.",
```

**Review by AsdarDevelops — COMMENTED** [review]
Added ES

**Review by Adri1 — APPROVED** [review]

**Review by CodeTappert — APPROVED** [review]

**Review by NicusPulcis — APPROVED** [review]
italian approved after screenshots

**Review by josericardo-fo — APPROVED** [review]

**Review by frutescens — APPROVED** [review]

**Review by KimJeongSun — APPROVED** [review]
LGTM

**Pull request merged**